plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1087,29 +1087,133 @@ public void testCaseInsensitivePredicate()
     }
 
     @Test
-    public void testArithmeticModuloPredicatePushdown()
+    public void testArithmeticPredicatePushdown()
     {
         if (!hasBehavior(SUPPORTS_PREDICATE_ARITHMETIC_EXPRESSION_PUSHDOWN)) {
-            assertThat(query("SELECT shippriority FROM orders WHERE shippriority % 4 = 0")).isNotFullyPushedDown(FilterNode.class);
+            assertThat(query("SELECT nationkey, name, regionkey FROM nation WHERE nationkey + 1 = 7"))
+                    .isNotFullyPushedDown(FilterNode.class);
+            assertThat(query("SELECT nationkey, name, regionkey FROM nation WHERE nationkey - 1 = 7"))
+                    .isNotFullyPushedDown(FilterNode.class);
+            assertThat(query("SELECT nationkey, name, regionkey FROM nation WHERE nationkey * 1 = 7"))
+                    .isNotFullyPushedDown(FilterNode.class);
+            assertThat(query("SELECT nationkey, name, regionkey FROM nation WHERE nationkey / 1 = 7"))
+                    .isNotFullyPushedDown(FilterNode.class);
+            assertThat(query("SELECT nationkey, name, regionkey FROM nation WHERE nationkey % 20 = 7"))
+                    .isNotFullyPushedDown(FilterNode.class);
+            assertThat(query("SELECT nationkey, name, regionkey FROM nation WHERE -(nationkey) = -7"))
+                    .isNotFullyPushedDown(FilterNode.class);
```

Comment:
Obviously verifying none of them work is overkill - IMO using just `+` or `*` should be fine? WDYT?

Reply:
yeah, keep it short.